src/commands/anime/studioghibli/characterlist.js

```diff
@@ -0,0 +1,28 @@
+const { Command, SwitchbladeEmbed } = require('../../')
+
+const fetch = require('node-fetch')
+
+module.exports = class CharacterList extends Command {
+  constructor (client) {
+    super(client, {
+      name: 'characterList',
```

Comment:
No uppercase characters in command names.
```suggestion
      name: 'characterlist',
```